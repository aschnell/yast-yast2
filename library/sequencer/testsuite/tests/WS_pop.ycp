{

include "testsuite.ycp";
include "./../src/sequencer.ycp";

TEST(``(WS_pop(nil)), [], nil);
TEST(``(WS_pop([])), [], nil);
TEST(``(WS_pop([1])), [], nil);
TEST(``(WS_pop([1,3])), [], nil);
TEST(``(WS_pop([1,3,5,7,9,11])), [], nil);

}
