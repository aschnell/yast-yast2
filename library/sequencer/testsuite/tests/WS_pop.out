Return	nil
Return	nil
Return	nil
Return	[[1], 1]
Return	[[1, 3, 5, 7, 9], 9]
